deployment.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: appsignal-kubernetes
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: appsignal-kubernetes
+  template:
+    metadata:
+      labels:
+        app: appsignal-kubernetes
+    spec:
+      serviceAccountName: appsignal-kubernetes-service-account
+      containers:
+      - name: appsignal-kubernetes
+        image: appsignal/appsignal-kubernetes:latest
+        env:
+        - name: APPSIGNAL_API_KEY
+          valueFrom:
+            secretKeyRef:
+              name: appsignal
+              key: api-key
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: appsignal-kubernetes-service-account
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: appsignal-kubernetes-role
+rules:
+  - apiGroups: [""]
+    resources:
+      - nodes
+      - "nodes/proxy"
+    verbs:
+      - get
+      - list
+      - watch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: appsignal-kubernetes-role-binding
+subjects:
+- kind: ServiceAccount
+  name: appsignal-kubernetes-service-account
+roleRef:
+  kind: Role
+  name: appsignal-kubernetes-role
+  apiGroup: rbac.authorization.k8s.io
```

Comment:
Since the `Role` has been replaced with `ClusterRole`, this `RoleBinding` can be deleted. Only the `ClusterRoleBinding` is required.